tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "module": "CommonJS",
```

Comment:
It's best if we can remove this and have the transpiler to read ES6 module format.

Reply:
Yep, will look into upgrading the transpiler.